src/Controls/src/Core/ImageBrush.cs

```diff
@@ -0,0 +1,34 @@
+﻿#nullable enable
+namespace Microsoft.Maui.Controls
+{
+	[ContentProperty(nameof(ImageSource))]
+	class ImageBrush : Brush
```

Comment:
This is internal for now, but after a few tests and some additional tweaks in other places, this will work for all things.